src/tests/multihost/admultidomain/test_multiforest.py

```diff
@@ -99,35 +96,28 @@ def test_0001_multiforest(multihost, newhostname, adjoin):
         multihost.client[0].service_sssd('start')
         time.sleep(10)
         # Search for the user in same forest and domain
-        getent_domain_user1 = multihost.client[0].run_command(
-            f'getent passwd user1@{ad_domain}', raiseonerr=False)
-        getent_domain_user2 = multihost.client[0].run_command(
-            f'getent passwd user2@{ad_domain}', raiseonerr=False)
-        id_domain_user1 = multihost.client[0].run_command(
-            f'id user1@{ad_domain}', raiseonerr=False)
-        id_domain_user2 = multihost.client[0].run_command(
-            f'id user2@{ad_domain}', raiseonerr=False)
+        getent_domain_user1 = multihost.client[0].run_command(f'getent passwd user1@{ad_domain}')
+        getent_domain_user2 = multihost.client[0].run_command(f'getent passwd user2@{ad_domain}')
+        id_domain_user1 = multihost.client[0].run_command(f'id user1@{ad_domain}')
+        id_domain_user2 = multihost.client[0].run_command(f'id user2@{ad_domain}')
         # Search for the user in a different forest and domain
-        getent_domain1_user1 = multihost.client[0].run_command(
-            f'getent passwd user1@{ad1_domain}', raiseonerr=False)
-        getent_domain1_user2 = multihost.client[0].run_command(
-            f'getent passwd user2@{ad1_domain}', raiseonerr=False)
-        id_domain1_user1 = multihost.client[0].run_command(
-            f'id user1@{ad1_domain}', raiseonerr=False)
-        id_domain1_user2 = multihost.client[0].run_command(
-            f'id user2@{ad1_domain}', raiseonerr=False)
+        getent_domain1_user1 = multihost.client[0].run_command(f'getent passwd user1@{ad1_domain}')
+        getent_domain1_user2 = multihost.client[0].run_command(f'getent passwd user2@{ad1_domain}')
+        id_domain1_user1 = multihost.client[0].run_command(f'id user1@{ad1_domain}')
+        id_domain1_user2 = multihost.client[0].run_command(f'id user2@{ad1_domain}')
 
-        multihost.client[0].run_command(restore_krb5, raiseonerr=False)
-        multihost.client[0].run_command(cleanup_krb5, raiseonerr=False)
+        multihost.client[0].run_command('mv /etc/hosts.bak /etc/hosts')
+        multihost.client[0].run_command(restore_krb5)
+        multihost.client[0].run_command(cleanup_krb5)
         client.restore_sssd_conf()
         client.clear_sssd_cache()
 
         # Evaluate test results
-        assert getent_domain_user1.returncode == 0
-        assert getent_domain_user2.returncode == 0
-        assert id_domain_user1.returncode == 0
-        assert id_domain_user2.returncode == 0
-        assert getent_domain1_user1.returncode == 0
-        assert getent_domain1_user2.returncode == 0
-        assert id_domain1_user1.returncode == 0
-        assert id_domain1_user2.returncode == 0
+        assert getent_domain_user1
```

Comment:
write a message to be written if the code returns False like,
assert getent_domain_user1, f"Could not find user1 {getent_domain_user1}!"